templates/dotnet/src/Appwrite/Enums/Enums.cs.twig

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace {{ spec.title | caseUcfirst }}.Enums
+{
+    public static class {{ enum.name | caseUcfirst | overrideIdentifier }}
+    {
+        {% for value in enum.enum %}
+            {% set key = enum.keys is empty ? value : enum.keys[loop.index0] %}
+        public static readonly string {{ key | caseUcfirst | replace({'-': ''}) }} => "{{ value }}"
+            {% if not loop.last %},{% endif %}
```

Comment:
```suggestion
            {% if not loop.last %}{% endif %}
```
Replaced the `,` with a `;` above, since these are [properties](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties)